### PR TITLE
Minor UI touches

### PR DESCRIPTION
**Pull request by dprokop**
This PR applies minor details pointed out by @oddlittlebird

### CODE REVIEW

**Review by torkelo — APPROVED** [review]
Nice! can you add another minor UI polish ? change viz picker grid layout to repeat(auto-fit,minmax(116px,1fr)) 

It breaks to single columns a bit too early right now

File: public/app/features/dashboard/panel_editor/VizTypePicker.tsx [hunk 1]
```diff
@@ -87,7 +87,12 @@ export const VizTypePicker: React.FC<Props> = ({ searchQuery, onTypeChange, curr
   return (
     <div className={styles.grid}>
       {hasResults ? (
-        renderList.map((plugin, index) => renderVizPlugin(plugin, index))
+        renderList.map((plugin, index) => {
+          if (plugin.state === PluginState.deprecated) {
```
Comment (tskarhed) [inline public/app/features/dashboard/panel_editor/VizTypePicker.tsx:91]: If you use a deprecated panel, will it be visible somewhere else?

**Review by tskarhed — APPROVED** [review]
LGTM, just one question below :)

**Comment by dprokop** [comment]
> Nice! can you add another minor UI polish ? change viz picker grid layout to repeat(auto-fit,minmax(116px,1fr))

@torkelo done

File: public/app/features/dashboard/panel_editor/VizTypePicker.tsx (hunk 1 above)
Comment (dprokop) [reply public/app/features/dashboard/panel_editor/VizTypePicker.tsx:91]: Nope

Comment (dprokop) [reply public/app/features/dashboard/panel_editor/VizTypePicker.tsx:91]: We need to come up with something here, good catch. In the meantime i'm gonna merge this

**Pull request merged**